test/system/uniswap-v2.test.js

```diff
@@ -60,14 +60,35 @@ describeFn("System -- swap signer bonds on UniswapV2", () => {
         expect(balance).to.equal(ethers.utils.parseEther("20"))
       })
     })
+
+    describe("signer bonds uniswap v2", () => {
```

Comment:
It looks like this test belongs to unit tests, because we're checking just a small functionality of approved "operators". Do you think we can create a unit test and move it there?

Reply:
Agreed, this should live in unit tests.

Reply:
@pdyraga @dimpar 
I moved the tests to unit tests [`SignerBondsUniswapV2.test.js`](https://github.com/keep-network/coverage-pools/blob/only-approved-signer-swap-operators/test/SignerBondsUniswapV2.test.js).
However, I skipped one testcase that was affected by this PR (the happy path for `swapSignerBondsOnUniswapV2`), as it's tested in a system test (it was hard to test it in unit tests, that's why we decided to write system tests for it). 

There are more simple functions in `SignerBondsUniswapV2` that could be tested in unit tests, but are unrelated to this PR (e.g. `setMaxAllowedPriceImpact`, `setSlippageTolerance`). Should I add them (in a separate PR)?